elections/2020/voters.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Knative TOC Election - 2020 Eligible Voters List"
+linkTitle: "2020 Voters List"
+weight: 30
+type: "docs"
+---
+
+
+# 2020 Technical Oversight Committee Election
+
+## Eligible Voters List
+
+The eligible voter process can be found in the [TOC election mechanics](../../mechanics/TOC.md) doc.
+
+* **2020-04-21** - Add initial voter eligability based on GitHub contributions
+
+---
+
+## GitHub IDs
+
+- Abd4llA
+- abrennan89
+- AceHack
+- adrcunha
+- ahmetb
+- akashrv
+- aliok
+- andrew-su
+- anniefu
+- antoineco
+- aslom
+- bbrowning
+- beemarie
+- capri-xiyue
+- cardil
+- chaodaiG
+- chizhg
+- coryrc
+- covbot
+- cppforlife
+- Cynocracy
+- daisy-ycguo
+- davidor
+- dgerd
+- dprotaso
+- dsimansk
+- duglin
+- dushyanthsc
+- ericlem
+- evankanderson
+- Fredy-Z
+- garron
+- grac3gao
+- grantr
+- grayside
+- greghaynes
+- Harwayne
+- hohaichi
+- houshengbo
+- ian-mi
+- igsong
+- itsmurugappan
+- jchesterpivotal
+- jcrossley3
+- jihuin
+- jmprusi
+- joaedwar
+- jonjohnsonjr
+- joshrider
+- JRBANCEL
+- k4leung4
+- kzjackson1
+- lberk
+- lionelvillard
+- liu-cong
+- markusthoemmes
+- mattmoor
+- mattmoor-sockpuppet
```

Comment:
I have deepest respect for the sockpuppet, but probably it should not vote this time :D

Reply:
Thought I removed it before I committed... Oops.